moveit_setup_assistant/templates/moveit_config_pkg_template/launch/gazebo.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="limited" default="false"/>
+  <arg name="paused" default="false"/>
+  <arg name="gui" default="true"/>
+  
+  <!-- startup simulated world -->
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" default="worlds/empty.world"/>
+    <arg name="paused" value="$(arg paused)"/>
+    <arg name="gui" value="$(arg gui)"/>
+  </include>
+
+  <!-- send robot urdf to param server -->
+  <param unless="$(arg limited)" name="robot_description" [URDF_LOAD_ATTRIBUTE] />
+
+  <param if="$(arg limited)" name="robot_description" [URDF_LOAD_ATTRIBUTE] />
```

Comment:
these two lines do the exact same thing, so the limited variable should be removed or the URDF_LOAD_ATTRIBUTE should change for some reason